msteams-platform/concepts/build-and-test/apps-localization.md

```diff
@@ -64,26 +68,29 @@ You will then want to add the 'localizationInfo' property with the default langu
 
-You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the 'localizationInfo' property of your manifest. Each file correlates to a language tag which the Teams client uses to choose the appropriate strings. The language tag takes the form of <language>-<region> but it is recommended to omit the <region> portion to target all regions that support the desired language.
+You can provide additional .json files with translations of all the user facing strings in your manifest. These files must adhere to the [Localization file JSON schema](../../resources/schema/localization-schema.md) and they must be added to the `localizationInfo` property of your manifest. Each file correlates to a language tag which the Teams client uses to select the appropriate strings. The language tag takes the form of `<language>-<region>` but it is recommended to omit the `<region>` portion to target all regions that support the desired language.
 
-The Teams client will apply the strings in this order: default language strings -> user's language only strings -> user's language + user's region strings.
+The Teams client applys the strings in the following order:
+default language strings -> user's language only strings -> user's language + user's region strings.
 
-For example, you provide a default language of 'fr' (French, all regions), and additional language files for 'en' (English, all regions) and 'en-gb' (English, Great Britain). If the user's language is set to 'en-gb':
+For example, if you provide a default language of `fr` (French, all regions), and additional language files for `en` (English, all regions) and `en-gb` (English, Great Britain), the user's language is set to `en-gb`. The following changes take place based on the language selection:
 
-1. The Teams client will take the 'fr' strings overwrite them with the 'en' strings.
-2. Overwrite those with the 'en-gb' strings.
+1. The Teams client takes the `fr` strings overwrite them with the `en` strings.
+1. Overwrite the `en` strings with the `en-gb` strings.
 
-If the user's language is set to 'en-ca': 
+If the user's language is set to `en-ca`, the following changes take place based on the language selection: 
 
-1. The Teams client will take the 'fr' strings overwrite them with the 'en' strings.
-2. Since no 'en-ca' localization is supplied, the 'en' localizations will be used.
+1. The Teams client takes the `fr` strings overwrite them with the `en` strings.
+1. Since no `en-ca` localization is supplied, the `en` localizations are used.
 
-If the user's language is set to 'es-es', the Teams client will take the 'fr' strings and will not override them with any of the language files.
+If the user's language is set to `es-es`, the Teams client takes the `fr` strings and does not override them with any of the language files.
```

Comment:
Similar to the previous paragraph, we should say why there is no override (= because no es or es-es translation was provided).